.vault-config/product-construction-dev.yaml

```diff
@@ -24,4 +24,4 @@ secrets:
       gitHubBotAccountSecret: 
         location: engkeyvault
         name: BotAccount-dotnet-bot
-      gitHubBotAccountName: dotnet-bot
+      gitHubBotAccountName: dotnet-bot
```

Comment:
I'd put the github secret here like it was in your branch

Reply:
Added, not sure why I removed it originally